Arrangement_on_surface_2/doc/Arrangement_on_surface_2/PackageDescription.txt

```diff
@@ -47,9 +47,7 @@ namespace ArrTraits {}
 /// \ingroup PkgArrangementOnSurface2Ref
 
 /*!
-  \code
-   #include <CGAL/draw_arrangement_2.h>
-  \endcode
+  `#include <CGAL/draw_arrangement_2.h>`
```

Comment:
what do you think of the following?

```
\#`include <CGAL/draw_arrangement_2.h>`
```

My main concern is missing warnings for 5.6 and 5.5 branches (+ I prefer a fully green page).
I know that my fix is not perfect as the # goes out of the `<code>` but honestly on my screen it does not make a big change (vertical alignment but really not a big deal). I can make the change if you agree and don't feel like making the change yourself.

Reply:
I'm not in favor of this type of coding things.
I think this is an issue / PR that should not be backported to the 5.5. and 5.6 branches , but when this is necessary I think it might be better to have an `ALIASES` for it like (I didn't test it):
- for the 1.9.6 and newer versions: ``cgalInclude{1}="`#include<%1>`"``
- for the 1.8.13 version: ``cgalInclude{1}="\#`include<%1>`"``

so that the nasty details are hidden.

The usage would be like `\cgalInclude{CGAL/draw_arrangement_2.h}`

Reply:
Due to our current infrastructure, we test release branches only with master (but once a week). So even without backporting we'd have the issue. I like the macro idea.

Reply:
So the actual problem is a problem with the infrastructure to be solved at another level ...

With "release braches" you mean the 5.5 and 5.6 branches where the problem without backporting won't be.

Reply:
The infrastructure problem won't be solve any time soon to be honest. That would mean several testsuite per day. But the macro is nice cause it shows how it should be done.

Reply:
yes release branches are 5.5 and 5.6 branches